packages/experimental/Menu/src/MenuTrigger/useMenuTrigger.ts

```diff
@@ -32,5 +50,15 @@ export const useMenuTrigger = (_props: MenuTriggerProps) => {
     setOpen(e, !open);
   };
 
-  return { onClick, onHoverIn, onHoverOut, componentRef: triggerRef, delayHoverIn: delayHover, delayHoverOut: delayHover };
+  return {
+    onClick,
+    onHoverIn,
+    onHoverOut,
+    componentRef: triggerRef,
+    delayHoverIn: delayHover,
+    delayHoverOut: delayHover,
```

Comment:
I don't think we want a delayHoverOut for macOS. If this is a requirement for win32, could we separate the two variables?